packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -19,7 +19,7 @@ export const latencyThreshold = 5000;
 
 class OpPerfTelemetry {
     private pongCount: number = 0;
-    private socketLatency = 0;
+    private lastPingLatency = 0;
```

Comment:
Initialize to NaN or something to avoid confusion if we measure an op before a ping?

Reply:
I actually don't know what Kusto will do with NaN. Probably undefined is better and just omit it in that case.

Reply:
Yep, please use undefined